client/src/pages/Home/index.js

```diff
@@ -0,0 +1,57 @@
+import React, { Component } from 'react';
+
+import { Search, ImageCarousel } from '../../components/index';
+import fakeImages from '../../components/ImageCarousel/fakeData';
+import plantIcon from '../../assets/images/icon-sign.svg';
+
+import './style.css';
+
+class Home extends Component {
+  // eslint-disable-next-line no-unused-vars
+  getSitSpots = data => {
+    // move to the searchResult route and send search creteria to it
+  };
+
+  render() {
+    return (
+      <>
```

Comment:
@SajedaIsmail I think it might be helpful to add a div as the top-most element instead of a fragment, because then you can give it a className (for example `className="home"`) and then use that in your css, to make your selectors more specific.  For example, in your css you could then change this:
`.top-container`
into this:
`.home .top-container`